core/common/enum.hpp

```diff
@@ -5,9 +5,20 @@
 
 #pragma once
 
+#include <array>
+#include <string_view>
 #include <type_traits>
+#include <boost/optional.hpp>
 
 namespace fc::common {
+  template<typename Enumeration, size_t Number>
+  using ConversionTable = std::array<std::pair<Enumeration, std::string_view>, Number>;
+
+  template <typename T>
+  auto& conversion_table() {
```

Comment:
```suggestion
  auto &conversion_table() {
```
Please, use auto-format for the file.